src/Education/beta/examples/Update-MgBetaEducationClassAssignmentSetting.md

```diff
@@ -41,3 +41,19 @@ Update-MgBetaEducationClassAssignmentSetting -EducationClassId $educationClassId
 ```
 This example shows how to use the Update-MgBetaEducationClassAssignmentSetting Cmdlet.
 
+### Example 3: Code snippet
+
+```powershell
+
+Import-Module Microsoft.Graph.Beta.Education
+
+$params = @{
+	"gradingCategories@delta" = @(
```

Comment:
This empty payload doesn't seem very useful as an example...

Reply:
Thanks for noticing this. I will go ahead and close this. However, there is a [fix](https://github.com/microsoftgraph/microsoft-graph-devx-api/pull/2001)for this issue. Hopefully, it will pick up the truncated payload the next time the pipeline runs.